src/models/generators/melodyDetector.coffee

```diff
@@ -1,6 +1,7 @@
 AubioPitch = include("lib/aubioPitch")
 Melody = include("models/melody")
-readDir = require("fs").readdirSync
+promisify = require("bluebird").promisifyAll
```

Comment:
Sip, lo hablamos en https://github.com/rodri042/scoremonkey-server/pull/4#diff-for-comment-29989185